components/AboutDedi/index.vue

```diff
@@ -0,0 +1,321 @@
+        },
+        {
+          tahap: 'Desa Digital 3.0',
+          title: 'Pelatihan Pemasaran Digital',
+          desc: 'Membangkitkan daya saing desa melalui pelatihan digital marketplace bagi pelaku Usaha Mikro Kecil Menengah (UMKM) dan Badan Usaha Milik Desa (BUMNDes).',
+          image: require('~/assets/images/about/tahap3.png')
+        },
+        {
+          tahap: 'Desa Digital 4.0',
+          title: 'Implementasi Pemanfaatan Teknologi',
+          desc: 'Meningkatkan produktifitas dan kemandirian masyarakat desa dengan meluncurkan ragam desa tematik sesuai dengan potensi desa. Tematik yang sudah berjalan diantaranya:',
+          image: require('~/assets/images/about/tahap4.png')
+        }
+      ]
+    }
+  },
+  mounted () {
+    this.content = this.contents[this.currIndex]
+  },
+  methods: {
+    prevSlide () {
+      if (this.currIndex > 0) {
+        this.currIndex--
+      }
+      this.content = this.contents[this.currIndex]
+    },
+    nextSlide () {
+      if (this.currIndex < this.contents.length - 1) { this.currIndex++ }
+      this.content = this.contents[this.currIndex]
+    }
+  }
+}
+</script>
+
+<style lang="postcss" scoped>
+.slide-fade-enter-active {
+  transition: all .3s ease;
+}
+.slide-fade-leave-active {
+  transition: all .5s cubic-bezier(1.0, 0.5, 0.8, 1.0);
+}
+.slide-fade-enter, .slide-fade-leave-to {
+  opacity: 0;
+}
+
+.about-dedi {
+
+  &__card {
+    box-shadow: 0px 4px 28px rgba(0, 71, 41, 0.08);
+    @apply font-serif font-bold bg-white rounded-[20px]
+    flex justify-between items-center;
+  }
+
+  &__info {
+    @apply py-6 px-4 w-full flex-1 xl:(p-[48px]);
+
+    &-title {
+      @apply text-blue-gray-800 font-bold text-[28px] leading-[42px] text-center
+      xl:(text-[32px] leading-[50px] text-left);
+    }
+
+    &-subtitle {
+      @apply mt-6 font-serif font-bold text-green-700 text-[24px] leading-[42px] text-center
+      xl:(text-[28px] leading-[60px] text-left);
+    }
+
+    &-title-desc {
+      @apply font-roboto text-blue-gray-500 text-center leading-[26px] mt-4 xl:(text-left);
+    }
+
+    &-desc {
+      @apply mt-3 font-sans font-normal text-blue-gray-400 leading-[26px] text-center xl:(text-left);
+    }
+
+    &-box-pagination {
+      @apply flex justify-between items-center mt-8 xl:(hidden);
+    }
+
+    &-pagination-dot {
+      @apply float-left mr-[20px] w-3 h-3 rounded-full bg-gray-300 transition-all duration-300 ease-in-out;
+
+      &--active {
+        @apply w-[44px] cursor-auto bg-green-600;
+      }
+    }
+
+    &-pagination-dot:last-child {
+      @apply mr-[unset];
+    }
+
+    &-btn-left {
+      @apply w-[42px] h-[42px] bg-green-700 rounded-full flex items-center justify-center cursor-pointer;
+
+      &--disabled {
+        @apply bg-gray-50;
+      }
+    }
+
+    &-btn-right {
+      @apply w-[42px] h-[42px] bg-green-700 rounded-full flex items-center justify-center cursor-pointer;
+
+      &--disabled {
+        @apply bg-gray-50;
+      }
+    }
```

Comment:
minor, to be discussed:
there're many duplicate code, can it simplify like this
```
    &-btn-left, &-btn-right {
      @apply w-[42px] h-[42px] bg-green-700 rounded-full flex items-center justify-center cursor-pointer;
      &--disabled {
        @apply bg-gray-50;
      }
    }
```

also for classes `arrow-left`, `arrow-right`, `icon-left`, `icon-right`, etc.

Reply:
i right thanks @yoslie  it more be simple